<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DgTerrainEditor</class>
 <widget class="QDialog" name="DgTerrainEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>450</y>
     <width>164</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QGraphicsView" name="view_current">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="view_new">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_current">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_preview">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>90</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="button_load">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>240</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Load New</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_select">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="list_current">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>151</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="list_new">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>151</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_apply">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>270</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DgTerrainEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DgTerrainEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
